app/cli/auth/account.go

```
@@ -91,8 +67,10 @@ func SelectOrSignInOrCreate() error {
 		return fmt.Errorf("error selecting account: account not found")
 	}
 
-	setAuth(&types.ClientAuth{
-		ClientAccount: *selected,
+	selectedAuth := *selected
+
+	setAuth(&shared.ClientAuth{
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)